File: 프로그래머스/Lv.0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        String numStr = Integer.toString(num);
        for (int i = 0; i < numStr.length(); i++) {
            if (numStr.charAt(i)==(char)('0'+k) ) {
                return i+1;
            }
        }
        return -1;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.js
function solution(num, k) {
    let arr = num.toString().split("");  
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] == k.toString()) {  
            return i + 1;  
        }
    }
    return -1; 
}
